CMakeLists.txt

```diff
@@ -47,22 +50,28 @@ if (OSX)
 endif ()
 
 
-# no unknown pragmas: ODB compiler uses some pragmas that the regular compiler doesn't need
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wno-unknown-pragmas")
-# Needed to statically link ODB
-set (CMAKE_C_FLAGS " -DSQLITE_ENABLE_UNLOCK_NOTIFY -DSQLITE_OMIT_LOAD_EXTENSION")
-set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-if (NOT CMAKE_BUILD_TYPE)
-    set (CMAKE_BUILD_TYPE  "RelWithDebInfo" CACHE STRING "Choose the type of build, options are: None(CMAKE_CXX_FLAGS or CMAKE_C_FLAGS used) Debug Release RelWithDebInfo MinSizeRel." FORCE)
+if (WINDOWS)
```

Comment:
This block is a mix of different stuff, for instance:

* Directory inclusion is listed in lines 15-16
* Boost_USE_STATIC_LIBS is enabled for other operative systems in lines 43 and 49, and it is the same for all OS
* CMAKE_C_FLAGS is the same for all OS

So to sum up, this looks fine but need a bit of organization.

Reply:
I put them all in one block, else there would have been a lot of if-else blocks in our cmake. We could compare it with the other version of CMakeLists.txt, then we can discuss which way to keep.